app/src/main/res/layout/tag_item.xml

```diff
@@ -35,7 +36,8 @@
         android:layout_toRightOf="@id/tag_icon"
         android:maxLength="70"
         android:text="There is a big party that is going on at this place, chaeck this out!! Can't wait to do this OMG this is so cool..."
```

Comment:
Do we need this placeholder text?

Reply:
just for when I design the layout to see what it would look like. Will get rid of it